### PR TITLE
Add mockserver for test

### PR DESCRIPTION
**Pull request by lhy1024**
to solve #22

### CODE REVIEW

File: bench/bench.go [hunk 1]
```diff
@@ -70,7 +70,7 @@ func (s *scaleOut) Run() error {
 	}
 	s.t.addTime = time.Now()
 	for {
-		time.Sleep(time.Second)
+		time.Sleep(time.Second * 10)
```
Comment (ZenoTan) [inline bench/bench.go]: Maybe make it a variable?

File: test/helper.go [hunk 2]
```diff
@@ -0,0 +1,92 @@
+package test
+
+import (
+	"encoding/json"
+	"io"
+	"io/ioutil"
+	"log"
```
Comment (ZenoTan) [inline test/helper.go]: pingcap/log?

File: bench/bench.go (hunk 1 above)
Comment (lhy1024) [reply bench/bench.go]: or we will introduce a config file?

Comment (ZenoTan) [reply bench/bench.go]: Will it depend on data size or machine nums, I think check per second is ok?

Comment (lhy1024) [reply bench/bench.go]: In order to avoid the expansion does not take effect, I will add an additional number of checks later

**Review by ZenoTan — APPROVED** [review]

**Comment by ZenoTan** [comment]
LGTM

**Pull request merged**